eth/stagedsync/stage_mining_exec.go

```diff
@@ -95,7 +95,7 @@ func SpawnMiningExecStage(s *StageState, tx kv.RwTx, cfg MiningExecCfg, quit <-c
 	forceTxs := current.ForceTxs
 	noempty := true
 
-	stateReader := state.NewPlainStateReader(tx)
+	stateReader := state.NewPlainState(tx, current.Header.Number.Uint64(), nil)
```

Comment:
```suggestion
	stateReader := state.NewPlainState(tx, current.Header.Number.Uint64(), systemcontracts.SystemContractCodeLookup[cfg.ChainConfig.ChainName]))
```

`NewPlainState()` is normally called by using wrapper `CreateHistoryStateReader()` implemented in `rpchelper` module. Except for testing, the third argument `systemContractLookup` is considered(not `nil`). 

For generalization, we should add `systemContractLookup` argument.